src/chocolatey/infrastructure.app/commands/ChocolateyPackCommand.cs

```diff
@@ -104,5 +108,25 @@ public virtual bool may_require_admin_access()
         {
             return false;
         }
+
+        private void parse_properties(string input, IDictionary<string, string> properties)
+        {
+            input = input.remove_surrounding_quotes();
+            if (string.IsNullOrWhiteSpace(input)) return;
+
+            var tokens = input.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries);
+            foreach (var token in tokens)
+            {
+                var pair = token.Split(new[] { '=' }, 2, StringSplitOptions.RemoveEmptyEntries);
+                if (pair.Length != 2)
+                {
+                    throw new ArgumentException(
+                        string.Format("Ill-formed name=value pair in token: {0}", token),
+                        "properties");
+                }
+
+                properties.Add(pair[0], pair[1].remove_surrounding_quotes());
+            }
+        }

```

Comment:
Okay. I looked for prior art (I'm a fan of project codebase consistency - just ask my teammates when I review most of our PRs) but didn't think to look at `install`. I didn't see that in usage output, though. Obviously (I think) this isn't like `--params` which I'm assuming at face value just passes the string through to the main install program as-is. But looking at `ChocolateyInstallCommand.handle_additional_argument_parsing`, would all name/value pairs be passed as `--foo=1 --bar=baz`. I guess I could see that as an alternative but - at least for the current install command help - that doesn't seem as obvious. I could update the documentation for both `install` and `pack` if you'd rather go that way. I can queue up the other requested changes for now.